mat64/index_no_bound_checks.go

```diff
@@ -63,3 +65,45 @@ func (m *Vector) Set(r, c int, v float64) {
 func (m *Vector) set(r int, v float64) {
 	m.mat.Data[r*m.mat.Inc] = v
 }
+
+// At returns the element at row r and column c.
+func (t *Symmetric) At(r, c int) float64 {
+	if r >= t.mat.N || r < 0 {
+		panic(ErrRowAccess)
+	}
+	if c >= t.mat.N || c < 0 {
+		panic(ErrColAccess)
+	}
+	return t.at(r, c)
+}
+
+func (t *Symmetric) at(r, c int) float64 {
+	if t.mat.Uplo == blas.Upper {
```

Comment:
Just curious: why is this version of at() comparing Uplo and blas.Upper, while the other at() does not?